packages/victory-core/src/victory-util/selection.ts

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import { Collection, VictoryCommonProps } from "./index";
+import type { DomainPropType, ScaleXYPropType } from "../types/prop-types";
+
+// Private Functions
+
+function transformTarget(
+  target: number,
+  matrix: DOMMatrix,
+  dimension: "x" | "y"
+) {
+  const { a, d, e, f } = matrix;
+  return dimension === "y" ? d * target + f : a * target + e;
+}
+
+function getTransformationMatrix(svg: SVGElement): DOMMatrix {
+  return (svg as SVGGraphicsElement).getScreenCTM()!.inverse();
```

Comment:
I think there's a balance to be made between letting things fail vs returning a default.  In some cases, a default is helpful; in some, it causes a hard-to-diagnose bug downstream.  
In this scenario, returning an empty matrix would cause difficult-to-diagnose bugs, whereas the cast would result in a very clear `getScreenCTM is not a function` error.  So I think the cast is a decent solution.  A better solution would be to strongly-type the arg as `SVGGraphicsElement`.  Another option is to explicitly throw errors, like `if (!(svg instanceof SVGGraphicsElement)) throw new Error('helpful error message')`.